exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,37 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isBetween5and10(num) {
+  if (num >= 5 && num <= 10) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isShortName(num) {
+ if (num.length < 7) {
+   return true;
+ } else {
+   return false;
+ }
+}
+
+function startsWithD(num) {
+ if (num[0] == "D") {
+   return true;
+ } else {
+   return false;
+ }
+}
```

Comment:
Good work. You could make this more concise by just returning the result of the comparison, i.e. return num < 0
Also, you should use a different variable name for the functions that are being passed a name, i.e. name rather than num.